python-template/{{cookiecutter.placeholder_repo_name}}/tox.ini

```diff
@@ -0,0 +1,72 @@
+[tox]
+envlist = py{35,36,3738}
+
+[doc8]
+max-line-length = 120
```

Comment:
(opinion)

I wish this were shorter! But you are following the style guide correctly here, so there's nothing to change.